docs/resources/role_grants.md

```diff
@@ -32,6 +32,8 @@ resource "snowflake_role" "other_role" {
 }
 
 resource "snowflake_role_grants" "grants" {
```

Comment:
This change was recently merged... https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/383/files#diff-cafd3d35c9890667669bb5ce2863de6e2965954c8c8a345065a829933780d28aR34